libkbfs/keybase_service_util.go

```diff
@@ -36,6 +36,7 @@ func serviceLoggedIn(ctx context.Context, config Config, session SessionInfo,
 			log.CWarningf(ctx,
 				"Failed to enable existing journals: %v", err)
 		}
+		jServer.MakeFBOsForExistingJournals(ctx)
```

Comment:
Should this happen even if the above failed?

Reply:
Probably not, good point.